docs/extra-build-config.md

```diff
@@ -0,0 +1,183 @@
+# Optional build configuration
+
+There are a set of ways in which a user can influence different paramenters of
+the build. We list here the ones that are closely related to this BSP or
+specific to it. For the rest please check:
+<http://www.yoctoproject.org/docs/latest/ref-manual/ref-manual.html>
+
+## Compressed deployed files
+
+1. Overwrite IMAGE_FSTYPES in local.conf
+  * `IMAGE_FSTYPES = "tar.bz2 ext3.xz"`
```

Comment:
These should be intended properly, currently only one space, should be 4? Intending properly also seem to fix numbering (see below comment)

1. Overwrite IMAGE_FSTYPES in local.conf
     * `IMAGE_FSTYPES = "tar.bz2 ext3.xz"`
2. Overwrite SDIMG_ROOTFS_TYPE in local.conf
    * `SDIMG_ROOTFS_TYPE = "ext3.xz"`
3. Overwrite SDIMG_COMPRESSION in local.conf
     * `SDIMG_COMPRESSION = "xz"`